sessions/2018/july/Session_9.md

```diff
@@ -0,0 +1,124 @@
+# SOFTWARE ENGINEERING IN JAVA
+
+## Session 9 (07/08/2018)
+
+- TDD
+
+### Notes:
+
+#### TDD
+
+There exists 3 types of Test framework (terms can vary): *Unit*, *component* and *integration*.
```

Comment:
3 types of tests - the number of frameworks (the libraries etc. that execute the tests on your machine) vary by language. In Java, there's really only one that's used: JUnit.